DirectProgramming/C++SYCL_FPGA/ReferenceDesigns/svd/src/non_std_convariance.hpp

```diff
@@ -0,0 +1,198 @@
+#ifndef __NON_STD_COVARIANCE_MATRIX_HPP__
```

Comment:
You should instead edit the `streaming_covariance_matrix.hpp` header file to have a new template parameter that allows standardized/non standardized covariance computations. Here this is mostly a copy/paste of the header right?